.github/workflows/coverity.yml

```diff
@@ -0,0 +1,36 @@
+---
+# vi: ts=2 sw=2 et:
+
+name: Coverity
+
+on:
+  schedule:
+    # Run Coverity daily at midnight
+    - cron:  '0 0 * * *'
+  pull_request:
```

Comment:
I'm not sure secrets are available when PRs are opened. If they were it would be possible for anyone to "steal" them by opening PRs and sending them somewhere.

Reply:
Yeah, this is mostly for maintainers (at least I do this in other repos) - if you open a PR from a branch directly in the upstream repo the secrets are accessible, so it makes debugging or changing the configuration less of a PITA. But in this case the action will fail even after the secrets are set up, since it's opened from a fork.

Reply:
> it makes debugging or changing the configuration less of a PITA

Agreed. I think one option would be to special-case that by checking where PRs come from and skip it if they come from forks or, maybe, add a comment somewhere. Then again it should make it easier to spot PRs that should be reopened from branches if it fails loudly.